test/ui/controllers/book_details_spec.js

```diff
@@ -35,7 +35,7 @@ describe('Book Details Controller', function () {
     var copy = { id: 1 };
 
     var copyBookEndpPoint = apiEndpoint.concat('/copies/').concat(copy.id).concat('?projection=copyWithBookInline');
-
+    var copyLibraryEndPoint = apiEndpoint.concat('/libraries/search/findBySlug?slug=undefined');
```

Comment:
that's not quite the behaviour we want to mock, but it will work for now. We can improve on it later.